site/static/docs/5.3/assets/js/color-modes.js

```diff
@@ -59,7 +59,7 @@
 
   window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', () => {
     const storedTheme = getStoredTheme()
-    if (storedTheme !== 'light' || storedTheme !== 'dark') {
+    if (storedTheme !== 'light' && storedTheme !== 'dark') {
```

Comment:
`!a || !b` => `!(a && b)` i.e. original was equivalent to `!(storedTheme === 'light' && storedTheme === 'dark')` which would only be true for Erwin Schrodinger.